samples/bluetooth/direct_test_mode/Kconfig

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Direct Test Mode sample"
+
+config DTM_ANTENNA_COUNT
```

Comment:
I wonder if pins configuration can be done better than with Kconfig ?

Reply:
This looks like a job for the `/zephyr,user` devicetree node support which @pabigot added. You can put miscellaneous GPIO pins and random values in there.

https://docs.zephyrproject.org/latest/guides/dts/bindings.html#inferred-bindings

Reply:
Ok. I moved pins configuration to overlay with the `/zephyr, user` node

Reply:
@KAGA164 I can't see it, did you forget to add a new file?

Reply:
Yes. I forgot remove Kconfig